uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/CoreConstants.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 multipleMatchingActivitiesFound=Multiple matching activities have been found.
-activityNotFound=Activity not found.
+activityNotFound=The file could not be opened automatically. Please verify that the file has a supported format and that its editor is enabled in Settings -> Roles -> Permissions -> Editors.
```

Comment:
@jomarko I'm not sure if we can do this. This activity not found message is also used for not screens. 

i.e. 
<img width="657" alt="screenshot 2018-11-26 13 15 04" src="https://user-images.githubusercontent.com/531351/49034586-63f66480-f180-11e8-90d2-88f1496b6ffe.png">

$goToPlace('fsdfsd') on browser console.

Could you please revert this for the old message?

Reply:
@ederign hi, could you please explain me this `$goToPlace('fsdfsd')` more? How user can invoke this, via browser? append it to url? r via browser console invoke some jquery?

However I am not for reverting, I think it shouldn't be so hard to find general enough message, for example:
```
Location you try to open is inaccessible. Please double check it is an existing location and you have permissions to see it (Settings -> Roles ...).
```

What do you think? I think it is better to provide more information to the user. I think `Activity not found` doesn't explain the problem enough.

Reply:
Agree @jomarko 	can you please update the PR?